locales/en-GB.yml

```diff
@@ -596,19 +596,19 @@ en-GB:
     filter_by:
     filters:
       air_date:
-      alphabetical:
+      alphabetical: Alphabetical
       date_added:
       date_rated:
       first_air_date:
-      most_popular:
+      most_popular: Most Popular
       my_rating:
-      newest:
+      newest: Newest
```

Comment:
Some of these strings are just duplicates of the values in `en-US`, which is the default. Can you remove all of the changes to this file except the `refresh_from_youtube` key from this PR?